src/Orangebeard.SpecFlowPlugin/ClientExecution/Logging/LogScope.cs

```diff
@@ -0,0 +1,37 @@
+﻿//using Orangebeard.Shared.Extensibility;
```

Comment:
we should remove unused imports

Reply:
I have removed unused imports across the project. I have just pushed this new version.